Source/WebKit/Shared/Cocoa/WKObject.h

```diff
@@ -28,6 +28,7 @@
 #import "WKFoundation.h"
 
 #import <type_traits>
+#import <wtf/AlignedStorage.h>
```

Comment:
Why do we need this include in this header? I'd expect it to be in other headers and maybe some implementation files.

Reply:
It was either that or adding the include to every file that was using API::ObjectStorage (and there were a lot). I took the easy way out and since all these files had to already include WKObject.h, I just added the include here.

I can go the hard way if you prefer though.

Reply:
I do prefer it, but not strongly enough to insist.

Reply:
I did it before landing.